src/jre.ts

```diff
@@ -39,11 +39,15 @@ export function url() {
 }
 
 export async function install(log: Logger) {
+  const jreIsAvailable = await isJreAvailable(log);
```

Comment:
@inverno different order of words. Can we directly use `await isJreAvailable(log)` in condition?